pyproject.toml

```diff
@@ -1,31 +1,25 @@
 [build-system]
-requires = [ "setuptools", "setuptools_scm" ]
+requires = ["setuptools", "setuptools_scm"]
```

Comment:
Don't know if we want to be pedantic about the versions that the docs describe:
`requires = ["setuptools>=64", "setuptools-scm>=8"]`